packages/utilities/psammead-storybook-helpers/src/withServicesKnob.jsx

```diff
@@ -34,6 +34,7 @@ export default ({
     script,
     locale,
     dir = 'ltr',
+    timezone = 'GMT',
```

Comment:
Setting a default timezone of 'GMT' since this is used when there is no clearly defined timezone for the service.